pkg/controllers/provisioning/v1alpha1/reallocation/controller.go

```diff
@@ -87,24 +90,39 @@ func (c *Controller) Reconcile(object controllers.Object) error {
 	}
 
 	// 4. Find Nodes Past TTL with Karpenter Labels
-	expired, err := c.filter.GetExpiredNodes(ctx, provisioner)
+	terminable, err := c.filter.GetTerminableNodes(ctx, provisioner)
 	if err != nil {
-		return fmt.Errorf("getting expired nodes, %w", err)
+		return fmt.Errorf("getting terminable nodes, %w", err)
 	}
-	if len(expired) == 0 {
+	if len(terminable) == 0 {
 		return nil
 	}
 
-	// 5. Cordon each node
-	if err := c.annotator.CordonNodes(ctx, c.filter.GetCordonableNodes(expired)); err != nil {
+	// 5. Cordon each node and label draining
```

Comment:
You don't need to leak the fact that the node is labeled in the comment. Consider this as an implementation detail of cordon.

Reply:
A lot of the other comments have `label ___` that help the reader understand the flow of the checks and annotations. I think it's important here to help the reader understand what's going on in the code without having to deep dive.